samples/x-iac-pipeline/README.md

```diff
@@ -0,0 +1,182 @@
+# Apigee X via IaC Automation Pipeline exposed in Multiple GCP Regions with Shared VPC and External HTTPS Load Balancer
+
+## Setup Instructions
+
+This sample deploys in a slightly different manner than you might be used to in case you have explored the other samples in this repository.
+
+Deploying this sample happens in two stages.
+1. As a first step the Terraform scripts in this directory will create a Bootstrap [GCP Project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) with all the resources required to deploy Apigee X with an Infrastructure as Code (IaC) Automation Pipeline.
+2. The second step will leverage two [Cloud Source Repositories](https://cloud.google.com/source-repositories) and [Cloud Build](https://cloud.google.com/build) for deploying Apigee X with the surrounding resources as well as a simple [httpbin](https://httpbin.org) proxy.
```

Comment:
maybe call out the names of the repos like 

```md
* **infra-repo** A repo that contains the terraform resources to provision Apigee X and the additional components.
* **app-rep** An Apigee example proxy repository.
```

Reply:
Thanks Daniel.
Added as suggested.